docs/ServerTraceContext.md

```diff
@@ -0,0 +1,32 @@
+# Server Trace Context
+
+## Rationale
+
+`splunk-rum.js` captures the server trace context from a `Server-Timing` header in the `traceparent` format.  Our
+server-side instrumentation generates this automatically.  This document describes how to recreate this if you want to 
```

Comment:
Our as in signalfx agents? Maybe we should mention that no support in otel as of yet.  We could also list the language agents that do support it if not all signalfx agents don't.

Reply:
Yes, I'll note that it's just the SignalFx agents rather than trying to keep a detailed list here.